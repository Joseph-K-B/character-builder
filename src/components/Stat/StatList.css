.catchphrase-list{
    color: white;
    background-color: #272838;
    /* background-color: #989FCE; */
    text-align: center;
}