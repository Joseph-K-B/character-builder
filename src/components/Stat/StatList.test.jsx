import { render, screen } from "@testing-library/react";
import StatList from "./StatsList";


it('should render unordered list of catchphrases', () => {
    const {container} = render(<StatList catchPhraseArr={['hello', 'goodbye']}/>);
    const list = screen.getByLabelText('catch-list');
    const listItems = screen.getAllByTestId('list-items');

    expect(list).toBeInTheDocument();
    expect(listItems).toBeDefined();
    expect(container).toMatchSnapshot();
});