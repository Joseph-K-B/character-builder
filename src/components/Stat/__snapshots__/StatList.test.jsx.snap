// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render unordered list of catchphrases 1`] = `
<div>
  <div>
    <ul
      aria-label="catch-list"
    >
      Catchphrases
    </ul>
    <li
      data-testid="list-items"
    >
      "
      hello
      "
    </li>
    <li
      data-testid="list-items"
    >
      "
      goodbye
      "
    </li>
  </div>
</div>
`;
